doc/source/analyzing/saving_data.rst

```diff
@@ -27,7 +27,7 @@ Geometric Data Containers
 -------------------------
 
 Data from geometric data containers can be saved with the
-:func:`~yt.data_objects.data_containers.save_as_dataset`` function.
+:func:`~yt.data_objects.data_containers.save_as_dataset` function.
```

Comment:
```suggestion
:func:`~yt.data_objects.data_containers.YTDataContainer.save_as_dataset` function.
```

Reply:
These are two different existing functions, the first one is correct

```
$ wxc yt.data_objects.data_containers.save_as_dataset
/Users/robcleme/dev/yt-project/yt/yt/frontends/ytdata/utilities.py:6
```